mappings/net/minecraft/util/thread/ThreadExecutor.mapping

```diff
@@ -1,30 +1,34 @@
-CLASS net/minecraft/class_1255 net/minecraft/util/ThreadExecutor
+CLASS net/minecraft/class_1255 net/minecraft/util/thread/ThreadExecutor
 	FIELD field_18318 name Ljava/lang/String;
 	FIELD field_18319 executionsInProgress I
-	FIELD field_5750 taskQueue Ljava/util/Queue;
+	FIELD field_5750 tasks Ljava/util/Queue;
 	FIELD field_5751 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD <init> (Ljava/lang/String;)V
 		ARG 1 name
-	METHOD method_16075 executeQueuedTask ()Z
+	METHOD execute (Ljava/lang/Runnable;)V
+		ARG 1 runnable
+	METHOD method_16075 executeTask ()Z
 	METHOD method_16211 createTask (Ljava/lang/Runnable;)Ljava/lang/Runnable;
 		ARG 1 runnable
 	METHOD method_18854 isOnThread ()Z
-	METHOD method_18855 clearTasks ()V
+	METHOD method_18855 cancelTasks ()V
 	METHOD method_18856 canExecute (Ljava/lang/Runnable;)Z
 		ARG 1 task
 	METHOD method_18857 executeTasks (Ljava/util/function/BooleanSupplier;)V
 		ARG 1 stopCondition
 	METHOD method_18859 executeTask (Ljava/lang/Runnable;)V
 		ARG 1 task
-	METHOD method_19537 executeSync (Ljava/lang/Runnable;)V
+	METHOD method_19537 submitAndJoin (Ljava/lang/Runnable;)V
 		ARG 1 runnable
-	METHOD method_20493 executeFuture (Ljava/lang/Runnable;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_20493 submit (Ljava/lang/Runnable;)Ljava/util/concurrent/CompletableFuture;
+		ARG 1 task
 	METHOD method_20813 waitForTasks ()V
-	METHOD method_21684 getTaskQueueSize ()I
+	METHOD method_21684 getTaskCount ()I
 	METHOD method_3777 getThread ()Ljava/lang/Thread;
-	METHOD method_5382 createFuture (Ljava/lang/Runnable;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_5382 submitAsync (Ljava/lang/Runnable;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 runnable
-	METHOD method_5383 executeQueuedTasks ()V
+	METHOD method_5383 executeTasks ()V
```

Comment:
maybe runTasks? execute(Runnable) is already being used above. The same applies to executeTask()